packages/release-brancher/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "./node_modules/gts"
+  }
```

Comment:
We should probably fix this layout issue in the generator library, so that we don't get this ugly red-x on all our new libraries -- better still, why can't our linter handle JSON?

Reply:
https://github.com/googleapis/repo-automation-bots/pull/1516